signup-service/src/main/resources/application-default.properties

```diff
@@ -146,11 +147,13 @@ mosip.signup.ui.config.key-values={\
 ## ----------------------------- Notification templates ----------------------------------------------------------------
 # Default charset encoding ISO-8859-1 does not support khmer language characters, so templates in khm language are base64 encoded.
 mosip.signup.sms-notification-template.send-otp.khm=4Z6U4Z+S4Z6a4Z6+IHtjaGFsbGVuZ2V9IOGeiuGevuGemOGfkuGelOGeuOGeleGfkuGekeGfgOGehOGeleGfkuGekeGetuGej+Gfi+GeguGejuGek+GeuCBLaElEIOGemuGelOGen+Gfi+GeouGfkuGek+GegOGflA==
+mosip.signup.sms-notification-template.send-otp.khm.encode=true
```

Comment:
Is it required to have a property for each template to check if the value is encoded?
can we create one property configured with the list of languages for which templates are encoded?